client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -580,7 +580,7 @@ class RequiredPluginsInstallView extends Component {
 			return (
 				<>
 					<SetupNotices />
-					<WoopLandingPage siteId={ siteId } startSetup={ this.startSetup } />
+					<WoopLandingPage siteId={ siteId } />
```

Comment:
We no longer need the `startSetup` prop since we're sending Atomic sites through the new /start/woocommerce-install flow.